webapp/interop_handler.go

```diff
@@ -18,10 +18,11 @@ type interopData struct {
 }
 
 // Set of years that are valid for Interop 20XX.
-var validYears = map[string]bool{"2021": true, "2022": true}
+var validYears = map[string]bool{"2021": true, "2022": true, "2023": true}
 
 // Year that any invalid year will redirect to.
-const defaultRedirectYear = "2022"
+// TODO(danielrsmith): Change this redirect for next year's interop page.
```

Comment:
Should create a bug for Interop 2024 that mentions the need to change this or some other artifact to remind future self that you need to change this. Not a blocker for this PR

Reply:
That's a good idea, as it's only capured in the code here. I'll make one!